deno/generate_website_screenshot/src/mod.ts

```diff
@@ -0,0 +1,29 @@
+import { base64Encode } from "./deps.ts";
+
+const getBase64FromUrl = async (url: string) => {
+  const data = await fetch(url);
+  if (data.ok) {
+    const blob = await data.blob();
+    const arr = await blob.arrayBuffer();
+    const base64String = base64Encode(arr);
+    return { success: true, screenshot: base64String };
+  } else {
+    return { success: false, screenshot: "" };
```

Comment:
If something was unsuccessful, there should be a descriptive message.

Reply:
resolved